custom-elements/connected-callbacks-html-fragment-parsing.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>Custom Elements: connectedCallback not called during HTML fragment parsing</title>
```

Comment:
This title and the assertion description is a bit misleading. `connectedCallback` is indeed called on elements parsed via fragment parsing. It's just that "synchronous custom elements flag" must be unset when creating an element during fragment parsing. Put it more pragmatically, "the HTML fragment parsing algorithm must not create a custom element synchronously" or more verbosely "the HTML fragment parsing algorithm must enqueue a custom element upgrade reaction instead of synchronously invoking its constructor"